core/declarations.gms

```diff
@@ -228,6 +228,14 @@ pm_share_CCS_CCO2(ttot,all_regi)                      "share of stored CO2 from
 
 pm_delta_histCap(tall,all_regi,all_te)                "parameter to store data of historic capacity additions [TW/yr]"
 
+p_prodSeReference(ttot,all_regi,all_enty,all_enty,all_te)         "read from gdx the output value in the reference run"
```

Comment:
I would change the description of these parameter to something like: "secondary energy in reference run", "final energy in reference run",...
Also, please add the units if possible to the parameter definition in between brackets.

Reply:
done